tests/setup.py

```diff
@@ -48,7 +48,7 @@ def run(self):
       license="apache2.0",
       packages=find_packages(),
       include_package_data=True,
-      install_requires=["ducktape==0.12.0", "requests==2.31.0", "psutil==5.7.2", "pytest==8.3.3", "mock==5.1.0"],
+      install_requires=["ducktape==0.12.0", "requests==2.32.4", "psutil==5.7.2", "pytest==8.3.3", "mock==5.1.0"],
```

Comment:
https://github.com/apache/kafka/blob/d6688f869cfa32f2fbb78ba25b9750445e85eb7d/.github/scripts/requirements.txt#L17

Our CI is still using version `2.32.3`. It is not a big deal, but updating to `2.32.4` would make things feel more consistent and comfortable :smiley:

Reply:
Yep! We should also review whether we should bump the versions of the actions we use. For example we use actions/checkout v4 while the latest is v5, actions/setup-python v5 while the latest is v6, etc ...